backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +687,128 @@ def test_cancel_workshop_attendance(self):
         self.assertIsNone(self.card.workshop_attendance)
         self.assertEqual(WorkshopAttendance.objects.count(), 0)
         self.assertEqual(self.card.status, AgileCard.READY)
+
+
+class ReviewerIdsSinceLatestReviewRequest(TestCase):
+
+    def setUp(self):
+        self.card_1 = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.card_2 = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.project_1 = self.card_1.recruit_project
+        self.project_2 = self.card_2.recruit_project
+        self.user = self.card_1.assignees.first()
+        self.assertIsNotNone(self.card_1.assignees)
```

Comment:
Another thing to consider: What is the purpose of the setUp?

Is the setup a test? And if it is not a test why does it have so many assertions?

It would be much cleaner to just have setup things happen in the setup function, then have all your test assertions in appropriate test functions. 

Are these assertions even necessary? They dont seem to have much to do with the functionality under test. So it's not immediately obvious what you are trying to do here.

Is this just sanity checking?

Also you are checking a few odd things, eg `self.assertIsNotNone(self.card_1.assignees)`.  I dont think that will ever be none because the any_card.assignees gives us a thing that we can use to create a queryset.

Eg even if the assignees was empty then `any_card_ever.assignees` would not be None.  Because `any_card_ever.assignees.all()` should always work and `None.all()` will error.

Can you please clean up your setup method. If any of these assertions are necessary we'll need to find a place for them. It would probably be good to remove most of them.